sdk/include/opentelemetry/sdk/trace/sampler.h

```diff
@@ -49,7 +49,19 @@ class Sampler
 {
 public:
   // TODO: Remove this placeholder with real class
-  class SpanContext;
+  class SpanContext
```

Comment:
Since this is just a mock class to get our samplers working for the time being, I think it's fine to leave this change as a part of this PR. If you think it should be separated into it's own PR, let me know and we can bring it up in our sync today.